modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectDisplayContext.java

```diff
@@ -45,11 +45,20 @@ public RedirectDisplayContext(
 	public List<NavigationItem> getNavigationItems() {
 		return NavigationItemListBuilder.add(
 			navigationItem -> {
-				navigationItem.setActive(!isShowRedirectNotFoundEntries());
+				navigationItem.setActive(isShowRedirectEntries());
 				navigationItem.setHref(_renderResponse.createRenderURL());
 				navigationItem.setLabel(
 					LanguageUtil.get(_httpServletRequest, "redirects"));
```

Comment:
- [ ] The current redirections in the redirections app should be available in an "Alias" tab.

Reply:
Done. See latest changes.